packages/manager/modules/pci/src/projects/project/storages/cold-archive/add/add.controller.js

```diff
@@ -39,7 +72,63 @@ export default class ColdArchiveConfigurationController {
     );
   }
 
+  isReadyForValidation() {
+    const { createOrLinkedMode } = this.userModel;
+    const { createMode, linkedMode } = this.userModel;
```

Comment:
**nitpick:** Two statements can be combined
```suggestion
    const { createMode, linkedMode, createOrLinkedMode } = this.userModel;
```

Reply:
done